examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI/Utils/Common.swift

```diff
@@ -0,0 +1,106 @@
+    static let uploadContainer: String! = "uploads"
+    static let videoContainer: String = "videos"
+    
+// NOTE: This connection string is a read-only SAS token, may need to be regenerated. Expires end of Aug
+    // swiftlint:disable line_length
+    static let sasConnectionString =
+        "BlobEndpoint=https://iosdemostorage1.blob.core.windows.net/;QueueEndpoint=https://iosdemostorage1.queue.core.windows.net/;FileEndpoint=https://iosdemostorage1.file.core.windows.net/;TableEndpoint=https://iosdemostorage1.table.core.windows.net/;SharedAccessSignature=sv=2019-12-12&ss=bfqt&srt=co&sp=rl&se=2021-08-30T07:04:27Z&st=2021-01-30T00:04:27Z&spr=https&sig=no%2BHSwTVjheowEdMSJ0iI6NdWvPXZ51n99PJG91O0ko%3D"
+}
+
+struct AppState {
+    static var application: MSALPublicClientApplication?
+
+    static var account: MSALAccount?
+
+    static let scopes = [
+        "https://storage.azure.com/.default"
+    ]
+
+    static var currentAccount: MSALAccount? {
+        if let account = AppState.account {
+            return account
+        }
+        guard let application = AppState.application else { return nil }
+        // We retrieve our current account by getting the first account from cache
+        // In multi-account applications, account should be retrieved by home account identifier or username instead
+        do {
+            let cachedAccounts = try application.allAccounts()
+
+            if !cachedAccounts.isEmpty {
+                return cachedAccounts.first
+            }
+        } catch let error as NSError {
+            print("Didn't find any accounts in cache: \(error)")
+        }
+        return nil
+    }
+
+    static var downloadOptions: DownloadBlobOptions {
+        let options = DownloadBlobOptions(
+            range: RangeOptions(calculateMD5: true)
+        )
+        return options
+    }
+
+    private static var internalBlobClient: StorageBlobClient?
+    static func blobClient(withDelegate delegate: StorageBlobClientDelegate? = nil) throws -> StorageBlobClient {
+        let error = AzureError.client("Unable to create Blob Storage Client.")
+        if AppState.internalBlobClient == nil {
+            guard let _ = AppState.application else {
+                fatalError("Application is not initialized. Unable to create Blob Storage Client.")
+            }
+            
+            let credential = StorageSASCredential(staticCredential: AppConstants.sasConnectionString)
+            AppState.internalBlobClient = try? StorageBlobClient(
+                endpoint: URL(string: "https://iosdemostorage1.blob.core.windows.net/")!,
+                credential: credential)
+        }
+        
+        guard AppState.internalBlobClient != nil else {
+            throw error
+        }
+
+        let client = AppState.internalBlobClient!
+        client.delegate = delegate
+        StorageBlobClient.maxConcurrentTransfers = 4
+        return client
+    }
+}
```

Comment:
Yes, this can pretty much all be removed and you can create the Blob client using the connection string initializer.